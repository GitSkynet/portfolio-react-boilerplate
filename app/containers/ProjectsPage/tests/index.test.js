import React from 'react';
import { shallow } from 'enzyme';

import { ProjectsPage } from '../index';

describe('<ProjectsPage />', () => {
  it('Expect to render and match the snapshot', () => {
    const getContent = jest.fn();
    const wrapper = shallow(<ProjectsPage getContent={getContent} />);
    expect(wrapper).toMatchSnapshot();
  });
});
