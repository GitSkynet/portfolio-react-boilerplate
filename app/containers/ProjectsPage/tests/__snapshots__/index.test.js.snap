// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectsPage /> Expect to render and match the snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <StyledArticle__StyledArticle>
    <HelmetWrapper
        encodeSpecialCharacters={true}
    >
        <title>
            Mijn projected
        </title>
        <meta
            content="Mijn projecten pagina van Johan Meester zijn portfolio"
            name="description"
        />
    </HelmetWrapper>
    <Styled(RowContainer)
        tagName="div"
    >
        <Breakpoints>
            <PageHeader
                isLeftHanded={true}
            >
                <FormattedMessage
                    defaultMessage="MY PROJECTS"
                    id="portfolio.page.projects.title"
                    values={Object {}}
                />
            </PageHeader>
        </Breakpoints>
    </Styled(RowContainer)>
    <Styled(RowContainer)
        tagName="div"
    >
        <Breakpoints>
            <ProjectsList__ProjectsList>
                
            </ProjectsList__ProjectsList>
        </Breakpoints>
    </Styled(RowContainer)>
</StyledArticle__StyledArticle>,
  "nodes": Array [
    <StyledArticle__StyledArticle>
      <HelmetWrapper
            encodeSpecialCharacters={true}
      >
            <title>
                  Mijn projected
            </title>
            <meta
                  content="Mijn projecten pagina van Johan Meester zijn portfolio"
                  name="description"
            />
      </HelmetWrapper>
      <Styled(RowContainer)
            tagName="div"
      >
            <Breakpoints>
                  <PageHeader
                        isLeftHanded={true}
                  >
                        <FormattedMessage
                              defaultMessage="MY PROJECTS"
                              id="portfolio.page.projects.title"
                              values={Object {}}
                        />
                  </PageHeader>
            </Breakpoints>
      </Styled(RowContainer)>
      <Styled(RowContainer)
            tagName="div"
      >
            <Breakpoints>
                  <ProjectsList__ProjectsList>
                        
                  </ProjectsList__ProjectsList>
            </Breakpoints>
      </Styled(RowContainer)>
</StyledArticle__StyledArticle>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ProjectsPage
        getContent={[Function]}
        projects={Array []}
/>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": ProjectsPage {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "getContent": [Function],
          "projects": Array [],
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <StyledArticle__StyledArticle>
          <HelmetWrapper
                    encodeSpecialCharacters={true}
          >
                    <title>
                              Mijn projected
                    </title>
                    <meta
                              content="Mijn projecten pagina van Johan Meester zijn portfolio"
                              name="description"
                    />
          </HelmetWrapper>
          <Styled(RowContainer)
                    tagName="div"
          >
                    <Breakpoints>
                              <PageHeader
                                        isLeftHanded={true}
                              >
                                        <FormattedMessage
                                                  defaultMessage="MY PROJECTS"
                                                  id="portfolio.page.projects.title"
                                                  values={Object {}}
                                        />
                              </PageHeader>
                    </Breakpoints>
          </Styled(RowContainer)>
          <Styled(RowContainer)
                    tagName="div"
          >
                    <Breakpoints>
                              <ProjectsList__ProjectsList>
                                        
                              </ProjectsList__ProjectsList>
                    </Breakpoints>
          </Styled(RowContainer)>
</StyledArticle__StyledArticle>,
        "_debugID": 2,
        "_renderedOutput": <StyledArticle__StyledArticle>
          <HelmetWrapper
                    encodeSpecialCharacters={true}
          >
                    <title>
                              Mijn projected
                    </title>
                    <meta
                              content="Mijn projecten pagina van Johan Meester zijn portfolio"
                              name="description"
                    />
          </HelmetWrapper>
          <Styled(RowContainer)
                    tagName="div"
          >
                    <Breakpoints>
                              <PageHeader
                                        isLeftHanded={true}
                              >
                                        <FormattedMessage
                                                  defaultMessage="MY PROJECTS"
                                                  id="portfolio.page.projects.title"
                                                  values={Object {}}
                                        />
                              </PageHeader>
                    </Breakpoints>
          </Styled(RowContainer)>
          <Styled(RowContainer)
                    tagName="div"
          >
                    <Breakpoints>
                              <ProjectsList__ProjectsList>
                                        
                              </ProjectsList__ProjectsList>
                    </Breakpoints>
          </Styled(RowContainer)>
</StyledArticle__StyledArticle>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ProjectsPage
    getContent={[Function]}
    projects={Array []}
/>,
}
`;
